examples/crm/src/dataProvider.ts

```diff
@@ -450,6 +484,23 @@ export const dataProvider = withLifecycleCallbacks(
                     },
                 };
             },
+            afterDelete: async (result, dataProvider) => {
+                const { data: company } = await dataProvider.getOne<Company>(
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏Could be a shared function ?